pkg/webhook/utils.go

```diff
@@ -60,17 +60,17 @@ func CreateVolumeMountForSecret(volumeName string, secret *core.Secret) corev1.V
 	}
 }
 
-func UpdateVolumeMounts(containers []corev1.Container, mount corev1.VolumeMount) []corev1.Container {
+func AppendVolumeMounts(containers []corev1.Container, mount corev1.VolumeMount) []corev1.Container {
 	res := make([]corev1.Container, 0, len(containers))
 	for _, c := range containers {
-		c.VolumeMounts = append(c.VolumeMounts, mount)
+		c.VolumeMounts = appendVolumeMountIfNotExists(c.VolumeMounts, mount)
 		res = append(res, c)
 	}
 
 	return res
 }
 
-func UpdateEnvVars(containers []corev1.Container, envVar corev1.EnvVar) []corev1.Container {
+func AppendEnvVars(containers []corev1.Container, envVar corev1.EnvVar) []corev1.Container {
```

Comment:
we have this method in pluginmachinery already right?

Reply:
Couldn't find that... can you post a link?

Reply:
Similar but different!
https://github.com/flyteorg/flyteplugins/blob/master/go/tasks/pluginmachinery/flytek8s/k8s_resource_adds.go#L118

Reply:
should we move them all to pluginmachinery?